File: Latihan_B/python/Ram.py
class Ram:
    # atribut private
    __capacity = ""
    __price = 0

    # constructor
    def __init__(self, capacity="", price=0):
        self.__capacity = capacity
        self.__price = price

    # method set untuk capacity dan price
    def setCapacity(self, capacity):
        self.__capacity = capacity
    def setPrice(self, price):
        self.__price = price

    # method get untuk capacity dan price
    def getCapacity(self):
        return self.__capacity
    def getPrice(self):
        return self.__price
    
    # method untuk output Ram
    def printRam(self):
        print("Ram")
        print("-------------")
        print("Capacity    : " + self.getCapacity())
        print("Price       : " + "Rp. " + str(self.getPrice()))

File: Latihan_A/Ram.cpp
class Ram
{
    private:
        // atribut
        string capacity;
        int price;

    public:
        // constructor
        Ram(){}
        Ram(string capacity, int price) {
            this->capacity = capacity;
            this->price = price;
        }

        // method set untuk capacity dan price
        void setCapacity(string capacity) {
            this->capacity = capacity;
        }
        void setPrice(int price) {
            this->price = price;
        }
        
        // method get untuk capacity dan price
        string getCapacity() {
            return this->capacity;
        }
        int getPrice() {
            return this->price;
        }

        // method output untuk Ram
        void printRam() {
            cout << "Ram" << endl;
            cout << "-------------" << endl;
            cout << "Capacity    : " << this->capacity << endl;
            cout << "Price       : " << "Rp. " << this->price << endl;
        }

        // destructor
        ~Ram(){}
};